data-config.json

```diff
@@ -14,6 +14,7 @@
         "js": true,
         "css": true,
         "html": true,
+        "generate-clientlib-textfiles": true,
```

Comment:
Let's change this to two properties for max flexibility and also to better match what we already have for option names:
```
"js": true,
"jstxt":true,
"css": true,
"csstext": true,
```

Reply:
I realize I accidentally put `csstext` here - I meant fo have `csstxt` (per my other comment)